__tests__/app.test.js

```diff
@@ -0,0 +1,33 @@
+const app = require("../app");
+const pool = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const testData = require("../db/data/test-data");
+const request = require("supertest");
+
+beforeEach(() => seed(testData));
+
+afterAll(() => {
+  pool.end();
+});
+
```

Comment:
At this stage in terms of error handling I might expect to see a not found path test in here